### DISCUSSION TITLE
使用webstorm开发代码提示感觉不友好，是我使用的姿势不对吗？

### DISCUSSION THREAD

**Discussion by MilesNext** [NervJS/taro#11720] [Q & A]
![image](https://user-images.githubusercontent.com/40167660/165090146-79119d7d-e90a-4b16-9294-655ca1878c04.png)
比如这里拿到请求res之后我想访问他的data属性，点击查看的时候跳出一堆，根本不知道点哪个进源码。

如果使用vscode则可以直接跳转到正确的源码部分。

请问各位大佬，是什么原因呢。

**Comment by shrekuu** [comment]
@monkeylmj  或许带上 TypeScript 会好些, 这样 IDE 不必乱猜你这个 res.data 是啥. 😅

**Reply by MilesNext** [reply]
> @monkeylmj 或许带上 TypeScript 会好些, 这样 IDE 不必乱猜你这个 res.data 是啥. 😅

请问是啥意思？ 什么叫带上TypeScript（新手

另外我感觉IDE应该能推断出具体是什么类型呀。 vscode已经推断的很正确了

**Reply by shrekuu** [reply]
带上TypeScript 就是给你的数据加上类型.
我刚下了一个最新的试了一下挺好的呀, 你也一个最新的用吧:

https://user-images.githubusercontent.com/3136026/165437302-b502cf8a-4f8a-4fb4-b633-d1a3cc112024.mp4

**Reply by MilesNext** [reply]
> 带上TypeScript 就是给你的数据加上类型. 我刚下了一个最新的试了一下挺好的呀, 你也一个最新的用吧:
> 
>  Untitled.mp4

感谢大佬耐心解答，试了最新版本的webstorm确实是没问题了~ 应该是2021版的webstorm还不够智能..
